### PR TITLE
add contribute models doc

### PR DESCRIPTION
**Pull request by Yancey0623**

### CODE REVIEW

File: doc/contribute_models.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# How to Contribute SQLFLow Models
+
+This guide will introduce how to contribute to SQLFlow models. If you haven't checked [Define SQLFLow Models](/doc/customized+model.md), please check it out first.
```
Comment (typhoonzero) [inline doc/contribute_models.md]: model developers may not need to see the design doc.

File: doc/contribute_models.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        `- mydnnclassifier.py
+    ```
+
+1. Define your model named `MyDNNClassifier`:
+
+    ``` python
+    import tensorflow as tf
+
+    class MyDNNClassifier(tf.keras.Model):
+    def __init__(self, feature_columns, hidden_units=[10,10], n_classes=2):
```
Comment (typhoonzero) [inline doc/contribute_models.md]: add indentation here.

File: doc/contribute_models.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+1. You can add a new mode definition Python script under the folder [sqlflow_models](/sqlflow_models). For example, adding a new Python script `mydnnclassfier.py`:
+
+    ``` text
+    `-sqlflow_models
+        |- dnnclassifier.py
+        `- mydnnclassifier.py
+    ```
+
+1. Define your model named `MyDNNClassifier`:
```
Comment (typhoonzero) [inline doc/contribute_models.md]: you can choose whatever name you like for your model. Your model definition should be a  [keras subclass model](https://keras.io/models/about-keras-models/#model-subclassing)

File: doc/contribute_models.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    ```
+
+1. Import `MyDNNClassfier` in [sqlflow_models/__ini__.py]:
+
+    ``` python
+    ...
+    from .mydnnclassfier import MyDNNClassifier
+    ```
+
+## Integration test with SQLFlow gRPC server
```
Comment (typhoonzero) [inline doc/contribute_models.md]: => Test Your Model

File: doc/contribute_models.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+    ``` bash
+    cd ./models
+    > docker run --rm -it -v $PWD:/models -p 8888:8888 sqlflow/sqlflow
+    ```
+
+1. Update `sqlflow_models` in the SQLFlow all-in-one Docker container:
+
+    ``` bash
+    > docker exec -it <container-id> pip install -U /models
```
Comment (typhoonzero) [inline doc/contribute_models.md:78]: `pip install -U` only checks the version, need to uninstall and install it in the container.

File: doc/contribute_models.md (hunk 1 above)
Comment (Yancey0623) [reply doc/contribute_models.md]: Polished the doc.

File: doc/contribute_models.md (hunk 2 above)
Comment (Yancey0623) [reply doc/contribute_models.md]: done.

File: doc/contribute_models.md (hunk 3 above)
Comment (Yancey0623) [reply doc/contribute_models.md]: thanks, done.

File: doc/contribute_models.md (hunk 5 above)
Comment (Yancey0623) [reply doc/contribute_models.md:78]: `pip install -U /models` works well, but throw an exception for the first execution:
``` text
    return self._has(path)
  File "/miniconda/envs/sqlflow-dev/lib/python3.6/site-packages/pip/_vendor/pkg_resources/__init__.py", line 1845, in _has
    return zip_path in self.zipinfo or zip_path in self._index()
  File "/miniconda/envs/sqlflow-dev/lib/python3.6/site-packages/pip/_vendor/pkg_resources/__init__.py", line 1722, in zipinfo
    return self._zip_manifests.load(self.loader.archive)
  File "/miniconda/envs/sqlflow-dev/lib/python3.6/site-packages/pip/_vendor/pkg_resources/__init__.py", line 1679, in load
    mtime = os.stat(path).st_mtime
FileNotFoundError: [Errno 2] No such file or directory: '/miniconda/envs/sqlflow-dev/lib/python3.6/site-packages/sqlflow_models-0.0.1-py3.6.egg'
```

I'm not sure what is the best practices, will test it later.

File: doc/contribute_models.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+    ``` python
+    import tensorflow as tf
+
+    class MyDNNClassifier(tf.keras.Model):
+        def __init__(self, feature_columns, hidden_units=[10,10], n_classes=2):
+            ...
+            ...
+    ```
+
+1. Import `MyDNNClassfier` in [sqlflow_models/__ini__.py]:
```
Comment (tonyyang-svail) [inline doc/contribute_models.md]: sqlflow_models/__ini__.py => `sqlflow_models/__init__.py`

File: doc/contribute_models.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+
+    ``` python
+    import tensorflow as tf
+
+    class MyDNNClassifier(tf.keras.Model):
+        def __init__(self, feature_columns, hidden_units=[10,10], n_classes=2):
+            ...
+            ...
+    ```
+
```
Comment (tonyyang-svail) [inline doc/contribute_models.md:35]: 1. You can test your  `MyDNNClassifier` by adding a new Python script `tests/test_mydnnclassifier.py` and run the test as `python tests/test_mydnnclassifier.py`.
    ```python
    from  sqlflow_models import MyDNNClassifier
    from tests.base import BaseTestCases
    
    import tensorflow as tf
    import unittest
    
    
    class TestMyDNNClassifier(BaseTestCases.BaseTest):
        def setUp(self):
            self.features = {...}
            self.label = [...]
            feature_columns = [...]
            self.model = MyDNNClassifier(feature_columns=feature_columns)
    
    
    if __name__ == '__main__':
        unittest.main()
    ```

File: doc/contribute_models.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+## Add an SQLFlow Model
+
+1. Open the [SQLFlow models repo](https://github.com/sql-machine-learning/models) on your web browser, and fork the official repo to your account.
+
+1. Clone the forked repo on your hosts:
+
+    ``` bash
+    > git clone https://github.com/<Your Github ID>/models.git
+    ```
+
```
Comment (tonyyang-svail) [inline doc/contribute_models.md:14]: 1. Set up your local python environment by `make setup && source venv/bin/activate`. If you are using PyCharm, you can simply `make setup` and then import the `models` folder as a new project.

**Review by tonyyang-svail — COMMENTED** [review]
Maybe include a couple more steps mentioned in the comments. :)

File: doc/contribute_models.md (hunk 6 above)
Comment (Yancey0623) [reply doc/contribute_models.md]: done.

File: doc/contribute_models.md (hunk 7 above)
Comment (Yancey0623) [reply doc/contribute_models.md:35]: thanks, done.

File: doc/contribute_models.md (hunk 8 above)
Comment (Yancey0623) [reply doc/contribute_models.md:14]: done.

**Review by tonyyang-svail — APPROVED** [review]
LGTM.

Maybe fix https://github.com/sql-machine-learning/models/pull/14#discussion_r302010582 in the next PR.

**Pull request merged**